app/helpers/taxonomy_helper.rb

```diff
@@ -10,10 +10,6 @@ def show_organization_tab?
     SETTINGS[:organizations_enabled] && User.current.allowed_to?(:view_organizations)
   end
 
-  def show_taxonomy_tabs?
```

Comment:
I think this should issue a deprecation warning and not be removed immediately, it's used by at least the Foreman discovery plugin: https://github.com/theforeman/foreman_discovery/blob/fe346a413d3a69fc9e878e768caea1119d055d10/app/views/discovery_rules/_form.html.erb#L31.